src/redux/reducers/experiments/experimentsUpdate.js

```diff
@@ -0,0 +1,16 @@
+import initialState from './initialState';
+
+const experimentUpdate = (state, action) => {
+  const { experimentId, experiment } = action.payload;
+
+  return {
+    ...initialState,
+    ...state,
+    [experimentId]: {
+      ...state[experimentId],
```

Comment:
check if this is ok and it won't fail if state for this experiment is empty

Reply:
check to see if experiment exists, if not return `null`